SimG4Core/PhysicsLists/interface/UrbanMscModel93.h

```diff
@@ -1,52 +1,29 @@
-//
```

Comment:
@civanch - in terms of removing the g4 license. what is your estimate of code overlap with the original UrbanMscMode93? It seems quite high to me.